### PR TITLE
counts vs commute network

### PR DESCRIPTION
**Pull request by juanfonsecaLS1**

### CODE REVIEW

File: README.Rmd [hunk 1]
```diff
@@ -245,10 +253,77 @@ ADF_dtype |>
 
 ```
 
+## Spatial Analysis
```
Comment (Robinlovelace) [inline README.Rmd:256]: Good to see this section.

File: README.Rmd [hunk 2]
```diff
@@ -245,10 +253,77 @@ ADF_dtype |>
...
+  st_as_sf(coords = c("longitude","latitude"),crs = 4326)
+sf_counts
+```
+
+
+
+A subset of the counts are taken based on a buffer of the `rnet_commute` object.
+
+```{r}
+rnet_buffer20 = rnet_commute |> st_buffer(dist = 20)
```
Comment (Robinlovelace) [inline README.Rmd:283]: That's reasonable. May be best to union the object first. How long did it take to run?

File: README.Rmd [hunk 3]
```diff
@@ -245,10 +253,77 @@ ADF_dtype |>
...
+
+sf_counts_selected = sf_counts[rnet_buffer20,]
+
+```
+
+### Approach A
+The nearest feature is joined to each point location
+```{r}
+sf_counts_joined = st_join(sf_counts_selected,rnet_commute,join = st_nearest_feature)
```
Comment (Robinlovelace) [inline README.Rmd:292]: :+1:

Comment (Robinlovelace) [reply README.Rmd:292]: As a starter for 10 in any case.

**Review by Robinlovelace — APPROVED** [review]
Great work Juan! Interesting results that we can iterate on.

File: README.Rmd (hunk 2 above)
Comment (juanfonsecaLS1) [reply README.Rmd:283]: The buffer takes 1.4 minutes to run on my machine. The rest of the code does not take more than 10 seconds.

Comment (Robinlovelace) [reply README.Rmd:283]: I guess it will be quicker if you do...

Comment (Robinlovelace) [reply README.Rmd:283]:
```suggestion
rnet_buffer20 = rnet_commute |> st_union() |> st_buffer(dist = 20)
```

1+ minute is a lot.

**Pull request merged**